src/libraries/Microsoft.Extensions.Hosting.Systemd/src/SystemdHelpers.cs

```diff
@@ -16,16 +16,15 @@ namespace Microsoft.Extensions.Hosting.Systemd
     /// </summary>
     public static partial class SystemdHelpers
```

Comment:
https://github.com/search?q=repo%3Adotnet%2Fruntime%20SystemdHelpers&type=code

from the search result, it seems no need to be `partial`, maybe this was introduced by mistake, we could remove the `partial` if we want, it may be a breaking change

while from the grep search result, there's no `partial` usage

https://grep.app/search?q=SystemdHelpers&filter[lang][0]=C%23

Reply:
Let's remove it. If everything still compiles, it was unnecessary.

(partial doesn't impact the resulting IL and it's not a breaking change to add/remove)

Reply:
https://github.com/dotnet/runtime/pull/62353/files#diff-8c7237d0f1cb42ad1f78b8c7172a3720c691024c1ea6ae995a6b76c5b1d584cc

find it, the reason why it becomes `partial` may because we used the `GeneratedDllImport` generator for it before, but now we refactored in 
https://github.com/dotnet/runtime/pull/63421/files#diff-8c7237d0f1cb42ad1f78b8c7172a3720c691024c1ea6ae995a6b76c5b1d584cc 

then we use `Interop.libc.GetParentPid()` instead while the `partial` reserved